CHANGELOG-2.0.md

```diff
@@ -0,0 +1,19 @@
+# Changelog since v1.2.0
+
+The version 2.0 is not compatible with v1.x. See Action Required section and update CSI driver manifests.
+
+## Action Required
+
+- The external-attacher now uses PATCH HTTP method to update API objects. Please update the attacher RBAC policy to allow the attacher to `patch`  VolumeAttachments and PersistentVolumes. See `deploy/kubernetes/rbac.yaml` for an example. ([#177](https://github.com/kubernetes-csi/external-attacher/pull/177), [@jsafrane](https://github.com/jsafrane))
+- The `-connection-timeout`, `-leader-election-type` and `-leader-election-identity` flags, deprecated in v1.2, have been removed. Please update your manifests for the external-attacher.
+  The `-dummy` flag has been removed. Please update your manifests for the external-attacher. ([#173](https://github.com/kubernetes-csi/external-attacher/pull/173), [@jsafrane](https://github.com/jsafrane))
+- Processing of ControllerUnpublish errors has changed. CSI drivers SHOULD return success (0), when a deleted node or volume implies that the volume is detached from the node. The external attacher treats NotFound error as any other error and it assumes that the volume may still be attached to the node. Please check behavior of your CSI driver and fix it accordingly. ([#165](https://github.com/kubernetes-csi/external-attacher/pull/165), [@jsafrane](https://github.com/jsafrane))
```

Comment:
should or must? should means that it's optional, but it sounds like detach will never succeed?

Reply:
"SHOULD" is defined here: https://www.ietf.org/rfc/rfc2119.txt and CSI refers to it.

Reply:
I see CSI uses "SHALL" (=MUST) instead of SHOULD, I'll fix this

Reply:
Fixed